.github/workflows/release-rc.yml

```diff
@@ -0,0 +1,151 @@
+      - name: Initialize public folder cache
+        id: public-cache
+        uses: actions/cache@v2
+        with:
+          path: public
+          key: public-cache-${{ hashFiles('pkg/webui/**', 'sdk/js/**/*.js', 'sdk/js/generated/*.json', 'config/webpack.config.babel.js', 'yarn.lock', 'sdk/js/yarn.lock')}}
+      - name: Initialize dll cache
+        if: steps.public-cache.outputs.cache-hit != 'true'
+        id: dll-cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            public/libs.*.bundle.js
+            .cache/dll.json
+          key: dll-cache-${{ hashFiles('yarn.lock') }}
+      - name: Initialize babel cache
+        id: babel-cache
+        uses: actions/cache@v2
+        if: steps.public-cache.outputs.cache-hit != 'true'
+        with:
+          path: .cache/babel
+          key: ${{ runner.os }}-babel-cache-${{ hashFiles('config/babel.config.json', 'config/webpack.config.babel.js') }}
+          restore-keys: |
+            ${{ runner.os }}-babel-cache-
+      - name: Build DLLs
+        if: steps.dll-cache.outputs.cache-hit != 'true'
+        run: tools/bin/mage js:buildDll
+        env:
+          WEBPACK_GENERATE_PRODUCTION_SOURCEMAPS: 'true'
+      - name: Build frontend
+        run: tools/bin/mage js:build
+        if: steps.public-cache.outputs.cache-hit != 'true'
+        env:
+          WEBPACK_GENERATE_PRODUCTION_SOURCEMAPS: 'true'
+      - name: Check for diff
+        run: tools/bin/mage git:diff
+      - name: Import the signing key
+        run: |
+          printf '%s' '${{ secrets.SIGNATURE_PASSPHRASE }}' >/tmp/gpg_passphrase
+          cat /tmp/gpg_passphrase | gpg --passphrase-fd 0 --no-tty --batch --import gpg_signing_key
+      - name: Run Goreleaser
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: 'v1.2.5'
+          args: release --config .goreleaser.snapshot.yml --snapshot --timeout 60m
+        env:
+          SIGN_KEY_NAME: ${{ secrets.SIGN_KEY_NAME }}
+      - name: Tag and Push Docker images
+        run: |
+          export RC_VERSION=${{ format('{0}-rc{1}', github.ref_name, github.event.inputs.rc_version) }}
+          export RC_VERSION=${RC_VERSION#release/}
+          docker tag lorawan-stack-dev:${{ github.sha }}-amd64 ${DOCKER_PREFIX}/lorawan-stack-dev:${RC_VERSION}-amd64
+          docker tag lorawan-stack-dev:${{ github.sha }}-arm64 ${DOCKER_PREFIX}/lorawan-stack-dev:${RC_VERSION}-arm64
+          docker push ${DOCKER_PREFIX}/lorawan-stack-dev:${RC_VERSION}-amd64
+          docker push ${DOCKER_PREFIX}/lorawan-stack-dev:${RC_VERSION}-arm64
+          docker manifest create ${DOCKER_PREFIX}/lorawan-stack-dev:${RC_VERSION} ${DOCKER_PREFIX}/lorawan-stack-dev:${RC_VERSION}-amd64 ${DOCKER_PREFIX}/lorawan-stack-dev:${RC_VERSION}-arm64
+          docker manifest push ${DOCKER_PREFIX}/lorawan-stack-dev:${RC_VERSION}
+        env:
+          DOCKER_CLI_EXPERIMENTAL: enabled
+          DOCKER_PREFIX: thethingsnetwork
```

Comment:
In the past we also pushed `latest` to GHCR (though not consistently; it's currently on 3.16.2), so perhaps we should also push to GHCR here?

Reply:
Fine with me.